Testreglar/2.1.1/App/app-2.1.1a-jbk.json

```diff
@@ -0,0 +1,164 @@
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "2.5"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva appside testar du?",
+			"ht": "<p>Oppgi appside-ID.</p>",
+			"type": "tekst",
+			"label": "Skjermbilde:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har appsida innhald/funksjonalitet det er mogleg å sveipe til?",
+			"ht": "<p>Sveip sakte for å navigere i appen.</p>\n<p> </p>\n<p> </p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appsida har ikkje innhald/funksjonalitet som er mogleg å nå/betene med sveiping."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Aktiver skjermlesar (opplesingsfunksjon).",
+			"ht": "<p>Slik testar du:</p>\n<ul>\n<li>Aktiver skjermlesar (opplesningsfunksjon)\n<ul>\n<li>iOS: <a href=\"https://support.apple.com/no-no/guide/iphone/iph3e2e415f/ios\">VoiceOver</a></li>\n<li>Android: <a href=\"https://support.google.com/accessibility/android/answer/6283677?hl=en\">Talkback</a></li>\n</ul>\n</li>\n<li>Opne appsida du testar</li>\n<li>Sjekk om det er mogleg å sveipe til minst eit element</li>\n</ul>",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			},
+			"type": "instruksjon"
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Får du til å nå alt innhald/funksjonalitet med sveiping?",
+			"ht": "<p>Sveip sakte for å navigere på appsida. Eksempel på innhald/funksjonalitet som skal vere mogleg å nå:</p>\n<ul>\n<li>Knappar</li>\n<li>Lenker/lenka bilder</li>\n<li>Skjemaelement</li>\n<li>Anna funksjonalitet</li>\n</ul>\n<p><strong>Merk:</strong> Eventuell sveipefelle, omlasting av appsida eller kontekstendring påvirkar ikkje om det mogleg å betene innhaldet.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G202"
+			],
+			"ruting": {
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.5"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Får du til å aktivere all funksjonalitet med dobbelttrykk?",
+			"ht": "<p>Prøv å aktivere alle lenker, knappar og funksjonar på appsida med dobbelttrykk.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G202"
+			],
+			"ruting": {
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"ja": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Alt innhald og funksjonalitet på appsida kan nåast med sveiping og aktiverast med dobbelttrykk."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Kva innhald/funksjonalitet fekk du ikkje til å nå med sveiping eller aktivere med dobbelttrykk?",
+			"ht": "<p>Beskriv innhaldet med få stikkord.</p>",
```

Comment:
Standard hjelpetekst for element er: "Beskriv elementet, slik at det er mogleg å identifisere det i ettertid. Viss det gjeld fleire element, registrerer du eit og eit."   Juster slik at den passer her.

Reply:
ok